composer.json

```diff
@@ -20,7 +20,7 @@
                 "version": "0.3.0-rc42",
                 "type": "drupal-library",
                 "dist": {
-                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-react/releases/download/release-develop/dist.zip",
+                    "url": "https://github.com/reload/dpl-react/releases/download/release-DDFLSBP-246-vi-skal-sorge-for-at-vi-har-defaults-pa-al-configuration-af-dpl-react-apps/dist.zip",
```

Comment:
Remember to remove this once both are approved.

Reply:
Yes 👍 
![image](https://github.com/danskernesdigitalebibliotek/dpl-cms/assets/998889/54f1341c-1cf3-4aed-93db-af895eefbd95)